libc/newhdrgen/yaml/sys/sys_time.yaml

```diff
@@ -1,8 +1,7 @@
 header: sys-time.h
 standards: Linux
 macros: []
-types:
-  - type_name: struct_timeval
```

Comment:
why was this removed?

Reply:
struct_timeval is already in the .h.def file, so becomes duplicated on the header file output